docs/book/src/development/development.md

```diff
@@ -138,6 +138,17 @@ You must configure systemd and iptables as described in https://kind.sigs.k8s.io
 
 You must install the `podman-docker` package to emulate the docker cli tool. However, this is not sufficient on its own as described below.
 
+### Configure slirp4netns to use a different CIDR
+
+By default we configure the controller and worker on 10.0.2.0/24. Rootless podman uses slirp4netns for networking, and unfortunately by default it uses the same CIDR internally. This prevents the management cluster from being able to route to the CI devstack.
```

Comment:
If we change the default CIDR, do we no longer need to configure it?

Reply:
Yes, we could certainly do that, too. It would just require updating these 3 values: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/b6695f2051feeebc674924cb71faf5a56cdb2ede/hack/ci/create_devstack.sh#L42-L44.

Incidentally, I did wonder if 10.0.2.0/24 was some kind of convention. If we changed it to something else, would it merit a comment explaining why it isn't 10.0.2.0/24?

Reply:
I am not sure why CIDR is 10.0.2.0/24. Let's update to 10.0.3.0/24 and see how it goes.